### ISSUE TITLE
geopandas.read.file - ValueError: set_wakeup_fd only works in main thread

### ISSUE THREAD

**Issue by DRMajhi** [spyder-ide/spyder#19159]
### What steps will reproduce the problem?

import geopandas
world = geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres'))

### Traceback
```python-traceback
Exception ignored in: <function BaseEventLoop.__del__ at 0x000002BDA2E0E940>
Traceback (most recent call last):
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\base_events.py", line 648, in __del__
    self.close()
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\proactor_events.py", line 679, in close
    signal.set_wakeup_fd(-1)
ValueError: set_wakeup_fd only works in main thread
Exception ignored in: <function BaseEventLoop.__del__ at 0x000002BDA2E0E940>
Traceback (most recent call last):
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\base_events.py", line 648, in __del__
    self.close()
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\proactor_events.py", line 679, in close
    signal.set_wakeup_fd(-1)
ValueError: set_wakeup_fd only works in main thread
Exception ignored in: <function BaseEventLoop.__del__ at 0x000002BDA2E0E940>
Traceback (most recent call last):
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\base_events.py", line 648, in __del__
    self.close()
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\proactor_events.py", line 679, in close
    signal.set_wakeup_fd(-1)
ValueError: set_wakeup_fd only works in main thread
Exception ignored in: <function BaseEventLoop.__del__ at 0x000002BDA2E0E940>
Traceback (most recent call last):
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\base_events.py", line 648, in __del__
    self.close()
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\proactor_events.py", line 679, in close
    signal.set_wakeup_fd(-1)
ValueError: set_wakeup_fd only works in main thread
Exception ignored in: <function BaseEventLoop.__del__ at 0x000002BDA2E0E940>
Traceback (most recent call last):
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\base_events.py", line 648, in __del__
    self.close()
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\proactor_events.py", line 679, in close
    signal.set_wakeup_fd(-1)
ValueError: set_wakeup_fd only works in main thread
Exception ignored in: <function BaseEventLoop.__del__ at 0x000002BDA2E0E940>
Traceback (most recent call last):
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\base_events.py", line 648, in __del__
    self.close()
  File "C:\Users\dmajhi\Anaconda3\envs\test\lib\asyncio\proactor_events.py", line 679, in close
    signal.set_wakeup_fd(-1)
ValueError: set_wakeup_fd only works in main thread
```

## Versions

* Spyder version: 5.2.2 
* Python version: 3.8.0
* Qt version: 5.15.2
* PyQt5 version: 5.15.7
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0;<8.0.0        :  7.31.1 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.4.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.4.4 (OK)
numpydoc >=0.6.0              :  1.4.0 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0                :  2.14.5 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.3.2;<1.4.0          :  1.3.3 (OK)
pylsp_black >=1.0.0           :  1.0.0 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.2.1;<5.3.0      :  5.2.2 (OK)
qtpy >=1.5.0                  :  2.0.1 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  63.4.1 (OK)
sphinx >=0.6.6                :  5.0.2 (OK)
spyder_kernels >=2.2.1;<2.3.0 :  2.2.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=17                      :  23.2.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  3.5.3 (OK)
numpy >=1.7                   :  1.23.2 (OK)
pandas >=1.1.1                :  1.2.5 (OK)
scipy >=0.17.0                :  None (NOK)
sympy >=0.7.3                 :  None (NOK)
```

**Comment by dalthviz** [comment]
Hi @DRMajhi thank you for the feedback! Seems like this is related with an issue from Python itself (and seems that was fixed for Python 3.8.2: https://github.com/python/cpython/issues/78860). Checking I found this SO post too: https://stackoverflow.com/a/60441568/15954282

So I would suggest you to create an env with a more recent Python version (I think the latest Python 3.8.x is Python 3.8.12)

**Comment by dalthviz** [comment]
Closing due to lack of response
